src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -1,122 +1,159 @@
+     *
+     * @param application
+     *            Application context.
+     */
+    private static void setupCommandTests(Application application) {
+        application.addTest("Load secret command", new SelectLoadSecretCommandOptionListener());
+
+        application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
+
+    }
+
+    /**
+     * Setup driver manager, and set default Job2dDriver for application.
+     *
+     * @param application
+     *            Application context.
+     */
+    private static void setupDrivers(Application application) {
+        Job2dDriver loggerDriver = new LoggerDriver();
+        DriverFeature.addDriver("Logger driver", loggerDriver);
+
+        DrawPanelController drawerController = DrawerFeature.getDrawerController();
+        Job2dDriver driver = new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic");
+        DriverFeature.addDriver("Line Simulator", driver);
+        DriverFeature.getDriverManager()
+                     .setCurrentDriver(driver);
+
+        driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
+        DriverFeature.addDriver("Special line Simulator", driver);
+        DriverFeature.updateDriverInfo();
+    }
+
+    private static void setupWindows(Application application) {
+
+        CommandManagerWindow commandManager = new CommandManagerWindow(CommandsFeature.getDriverCommandManager());
+        application.addWindowComponent("Command Manager", commandManager);
+
+        CommandManagerWindowCommandChangeObserver windowObserver = new CommandManagerWindowCommandChangeObserver(commandManager);
+        CommandsFeature.getDriverCommandManager()
+                       .getChangePublisher()
+                       .addSubscriber(windowObserver);
+    }
+
+    /**
+     * Setup menu for adjusting logging settings.
+     *
+     * @param application
+     *            Application context.
+     */
+    private static void setupLogger(Application application) {
+
+        application.addComponentMenu(Logger.class, "Logger", 0);
+        application.addComponentMenuElement(Logger.class, "Clear log", (ActionEvent e) -> application.flushLoggerOutput());
+        application.addComponentMenuElement(Logger.class, "Fine level", (ActionEvent e) -> logger.setLevel(Level.FINE));
+        application.addComponentMenuElement(Logger.class, "Info level", (ActionEvent e) -> logger.setLevel(Level.INFO));
+        application.addComponentMenuElement(Logger.class, "Warning level", (ActionEvent e) -> logger.setLevel(Level.WARNING));
+        application.addComponentMenuElement(Logger.class, "Severe level", (ActionEvent e) -> logger.setLevel(Level.SEVERE));
+        application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
+    }
+
+    private static void addMouseDrawing(Application app) {

```

Comment:
Please consider extracting this feature to src (outside test folder)
Too much changest in TestJobs2dApp.java